docs/src/technik/coding_conventions/db_naming.md

```diff
@@ -0,0 +1,18 @@
+# Uppercase-Convention für Datenbanken
+
+## Kontext
+
+Aktuell sind sowohl Namen als auch Properties der Datenbanktabellen mal in Uppercase, mal in Lowercase.
+Sofern Oracle und h2 nicht durch eine Änderung der migration-scripts auf Probleme stößt, wurde die Frage gestellt, die
+Tabellennamen und Properties zu vereinheitlichen.
+
+## Entscheidung
+
+Nachdem ein Renaming der migration-scripts gemäß Recherche möglich ist, werden die Skripte aller Services refactored:
+Syntax in Uppcase, Tabellenname in CamelCase und die Properties in lowercase.
+Beispiel:
+CREATE TABLE Wahlbezirk
+(
+id VARCHAR2(36) NOT NULL,
+wahlbezirkArt VARCHAR2(3)  NOT NULL
+)
```

Comment:
```suggestion

Beispiel:

```sql
CREATE TABLE Wahlbezirk
(
id VARCHAR2(36) NOT NULL,
wahlbezirkArt VARCHAR2(3) NOT NULL
)
```

Ergebnis sollte so aussehen, was es aus meiner Sicht besser lesbar macht:  
![grafik](https://github.com/user-attachments/assets/61a2184a-6a3b-45b5-a161-f3f15a0519b5)

Reply:
du könntest zu dem SQL-Code auch die Klasse mit anzeigen. Analog wie es bei dem Beispiel in den Namenskonventionen der Tests gemacht ist.

Dann sieht man SQL-Code und Java-Code was es noch verständlicher macht.